### ISSUE TITLE
Update Staging Review operational process due to new SR requirements

### ISSUE THREAD

**Issue by shiragoodman** [department-of-veterans-affairs/va.gov-team#53698]
## User Story
As a Governance team member, I want to make sure I understand what steps I should take if I notice a VFS team is not meeting the SR requirements so that Governance team can take consistent action each time.

Assignee: @allison0034 
Peer Reviewer: @CherylEvans 

## Description
Governance team will be introducing 5 new requirements for Staging Review to ensure VFS products are ready and able to be appropriately tested. Before we can roll the changes out to VFS teams, we need to determine how these new requirements will impact our operational process.

## Impacted Artifacts
[How to document feedback for Staging Review ](https://vfs.atlassian.net/wiki/spaces/PPS/pages/1839530098/How+to+document+feedback+for+Staging+Review) - new section "What to do if Staging Review Requirements are not met"
[On-support guidance](https://vfs.atlassian.net/wiki/spaces/PPST/pages/2454093924/On-Support+guide) - new section Cancelling a Staging Review if requirements are not met

## Tasks
- [x] Add new section to document in artifacts containing this info (wordsmith):
  - [x] If anyone on our team cannot complete the review, we need to reschedule. Unless there was a situation where QA (or whoever) was going to be able to complete by the meeting date/time and that individual agreed. So maybe we put something in our guidance that a discussion needs to happen within our practice areas in those situations. 
  - [x] Communicate to the VFS team on the Slack thread where the request is since we are going to start adding that to each touchpoint section in the GH ticket. We should tell them it is cancelled for "x" reason. Tell the VFS team they are responsible for rescheduling via Calendly.
  - [x] Completely cancel the invite. In the staging section of the GH ticket strikethrough the date and put cancelled.
  - [x] We repeat this process until they can successfully have a Staging review
- [x] Update on support guidance
  - [x] The on support team member should communicate to the VFS team on the original Slack thread request with: 
  - [x] Hi @handle (VFS person who made the request). Unfortunately, while completing testing for your Staging Review, we encountered a violation of the following Staging Review requirement:
enter Staging Review requirement
More specifically, ______ (include specific details from the Gov team).
Since we are blocked from completing testing, we are cancelling the Staging Review. Once you have resolved the requirements violation, you can reschedule by following the “VFS Actions” in the ticket and using the Staging Review Calendly calendar.
Please let us know if you have any questions.
  - [x] In the staging review section of the Collaboration Cycle request ticket, under Platform actions, put “Cancelled” then strikethrough the date/time. Add “Rescheduled:”.
Eg. Meeting date/time: Cancelled: Thursday, March 9, 2023 at 2:00PM Eastern Rescheduled :
  - [x] Open the invite on the Collaboration Cycle calendar and click delete event. In the optional message area add:
  - [x] Cancelled because 1 of the Staging Review requirements was not met. VFS team has received detailed information and guidance for next steps.
  - [x] Click Send.
  - [x] We repeat this process until they can successfully have a Staging review
- [ ] Once "What to do if Staging Review Requirements are not met" section has been drafted and peer reviewed, share with @shiragoodman and the rest of Governance team via #platform-governance-team members Slack channel for additional review. 

## Peer Review
_To be completed by peer reviewer_
- [x] User story and acceptance criteria are met

## Acceptance Criteria
- [x] We have a defined process to follow for identifying when SR requirements are not met
- [x] We have a defined process for communicating to the VFS teams that SR requirements are not met and for communicating next steps
- [x] We have a defined process for cancelling the SR

### Refinement
- [x] Ticket has user story, description, tasks, and acceptance criteria
- [x] Ticket has [additional appropriate labels](https://vfs.atlassian.net/wiki/spaces/PPST/pages/2220359751/WIP+Tech+Debt+Ticket+Organization), if applicable
- [x] Ticket has been sized 
- [x] Once all above items are checked, add the 'Ready' label.
### Planning
_If this ticket is picked up from the Backlog mid-sprint, connect with Shira to ensure the below items are completed correctly_
- [ ] Ticket has been discussed as a team at Planning
- [x] Ticket has been assigned to appropriate initiative or quarterly epic
- [x] Ticket has been assigned to a Sprint
- [x] Ticket has been moved to the Planned pipeline in the Governance team board
- [x] Ticket as an assignee and peer reviewer

**Comment by shiragoodman** [comment]
@allison0034 @CherylEvans I have a few concerns here. The 4 requirements that must be met are listed in the Solution/Phase 1 section of the [initiative ticket. ](https://app.zenhub.com/workspaces/platform-governance-team-5f85b91c14d8df0018fac414/issues/gh/department-of-veterans-affairs/va.gov-team/53070) Would it make sense that all 5 Governance team members need to be blocked by at least 1 of the requirements in order for us to reach out to cancel? Or are each of the 4 requirements on a case by base basis? For example, requirement #1 impacts everyone, but maybe requirement #4 re: false bugs only impacts 1 or 2 practice areas. 

Other questions: How do we communicate with the VFS that the Staging is cancelled? What messaging should we use to cancel? Do we actual cancel the Calendly event or just add the word "cancelled" to the title? What about rescheduling? Should the VFS team re-request their staging Review via Calendly or do they contact us to reschedule? What if Governance team is still experiencing the requirement not being met?

Basically, what does/should our internal process look like? How do we determine that the VFS team has not met the requirement and what is our process for communication to the VFS, cancelling the SR and what are the next steps we provide to the VFS?

IF you 2 want to take a shot at answering these questions, and putting forth a proposal of how this should be handled, please let me know. I'm a bit busy over the next 3 days, so it would be super helpful if you could get started on helping us figure this out.

**Comment by allison0034** [comment]
@shiragoodman 

> Would it make sense that all 5 Governance team members need to be blocked by at least 1 of the requirements in order for us to reach out to cancel? Or are each of the 4 requirements on a case by base basis? For example, requirement #1 impacts everyone, but maybe requirement #4 re: false bugs only impacts 1 or 2 practice areas

I think if anyone on our team cannot complete the review, we need to reschedule. Unless there was a situation where QA (or whoever) was going to be able to complete by the meeting date/time and that individual agreed. So maybe we put something in our guidance that a discussion needs to happen within our practice areas in those situations.

> Other questions: How do we communicate with the VFS that the Staging is cancelled? 

I prefer the Slack thread where the request is since we are going to start adding that to each touchpoint section in the GH ticket.

> What messaging should we use to cancel? Do we actual cancel the Calendly event or just add the word "cancelled" to the title? 

We should tell them it is cancelled for " " reason.

I would completely cancel the invite. If it is far enough out someone else may want a DI/MP. But in the staging section of the GH ticket, I would strikethrough the date and put cancelled incase we want to "keep track" of these situations.

> Should the VFS team re-request their staging Review via Calendly or do they contact us to reschedule?

They should pick a new date via Calendly, that way the ownership is not on us.

> What if Governance team is still experiencing the requirement not being met?

Then we do the same process. If we notice this is a reoccurring problem then we should come up with a solution, but I don't think it will be.

**Comment by CherylEvans** [comment]
@shiragoodman I agree with Allison 100%.  I think taking this approach will keep it from reoccurring. Once teams know that their timeline might be impacted, they will ensure they meet the requirements for the reviews prior to scheduling.

**Comment by shiragoodman** [comment]
thank you both for providing your feedback. I have assigned @allison0034 to this ticket with @CherylEvans as the peer reviewer. I updated the AC. I did not add specific tasks - only to share on Slack with our team once peer review is complete so we all can weigh in.

LMK if you have any questions.

**Comment by shiragoodman** [comment]
@allison0034 since Cheryl is out today, I'm going to take a look first and provide comments in order to move this along :)

**Comment by shiragoodman** [comment]
@allison0034 I provided comments. Please take a look and let me know if you have any questions!

**Comment by allison0034** [comment]
@CherylEvans I updated everything per Shira's comments, ready for your peer review : )

**Comment by CherylEvans** [comment]
@allison0034 I tweaked a few things for grammar and flow; otherwise, it looks great! :-)
